InfiniteRecharge/src/main/java/frc/robot/subsystem/vision/VisionConstants.java

```diff
@@ -2,19 +2,32 @@
 
 public class VisionConstants {
 
-    static final double TARGET_HEIGHT_INCHES = 89.75;//81.0;//82.25;//89.75;//98.25;//82.25;// 98.25;// for the absolute top part of the tape
+
+    // height of the target from the ground
+    // there are a bunch of different numbers since we were messing with this number to get the distance to work
+    static final double TARGET_HEIGHT_INCHES = 89.75;//ignore these-> //81.0;//82.25;//89.75;//98.25;//82.25;// 98.25;// for the absolute top part of the tape
+    // height of the camera from above the ground
     static final double CAMERA_HEIGHT_INCHES = 22.6;
+    // cemra mounting angle
     static final double CAMERA_MOUNTING_ANGLE = 30;
-    
+
+    // what angle the ball shoots at
+    // not used anywhere
+    // and also wrong? since we control the angle of the shot
     static final double BALL_SHOOTING_ANGLE = 30; // wild guess
 
+    // bool to use/not use running average filter for calculating the tx
     static final boolean USE_FILTER = false;
+    // how many values the filter should use
     static final int FILTER_LENGTH = 25;
 
+    // flag to use/not use the auto-pipeline-switching-to-adjust-zoom-based-on-distance feature
     static final boolean ENABLE_AUTO_ZOOM = false;
 
+    // We have a tx bias since the drive auto-align requires it
     static final double TX_BIAS_DEG = Math.PI; // about 3.1, so why not
 
+    // getteeeeeeeeeeeeeeers
```

Comment:
This getter is here because the variable itself is package private?

Reply:
yeah the method is used in Sim.java as well